### DISCUSSION TITLE
Changing default document language and spelling check rules

### DISCUSSION THREAD

**Discussion by firatciftci** [dolanmiu/docx#1655] [Q&A] [answered]
Hello! I am trying to change the language/localization of the .docx document that I am generating – it defaults to "English (United States)" (`'en-US'`), but I would like it to always be "Turkish (Turkey)" (`'tr-TR'`). Plus, the document does not need spelling or grammar check, so I would like that option to be turned off by default as well. Is there a way to achieve these parameters using this library? Thank you so much!

**Comment by dolanmiu** [comment, marked as answer]
Yup, this is now done! With this PR:

https://github.com/dolanmiu/docx/pull/1848

Example of how its done is here:

https://github.com/dolanmiu/docx/blob/master/demo/83-setting-languages.ts
